messagebus/src/vespa/messagebus/network/rpcnetwork.cpp

```diff
@@ -423,6 +423,8 @@ RPCNetwork::sync()
 void
 RPCNetwork::shutdown()
 {
+    // Unschedule any pending target pool flush task that may race with shutdown target flushing
+    _scheduler.Unschedule(_targetPoolTask.get());
```

Comment:
Kill is even better since it also protects against the task being scheduled again.

Reply:
Fixed